src/main/java/org/tron/common/runtime/vm/PrecompiledContracts.java

```diff
@@ -90,6 +108,12 @@ public static PrecompiledContract getContractForAddress(DataWord address) {
         public abstract long getGasForData(byte[] data);
 
         public abstract Pair<Boolean, byte[]> execute(byte[] data);
+
+        public static byte[] callerAddress;
```

Comment:
![Vulnerability](https://sonarsource.github.io/sonar-github/vulnerability.png 'Vulnerability') Vulnerability: Make this "public static callerAddress" field final ([squid:S1444](https://sonarcloud.io/organizations/tron-zhaohong/rules#rule_key=squid%3AS1444))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=java-tron&pullRequest=1172&issues=AWTGpeR3xXt_iLiSog5P&open=AWTGpeR3xXt_iLiSog5P)

[ISSUE_KEY]: AWTGpeR3xXt_iLiSog5P